int32 frequency                                     # frequency of interpolated trajectory
float32 max_vel                                     # max linear velocity
float32 max_acc                                     # max linear acceleration
float32 max_yawrate                                 # max angular velocity
float32 max_yawrate_dot                             # max angular acceleration
geometry_msgs/PoseArray poses                       # array of poses to interpolate
---
trajectory_msgs/MultiDOFJointTrajectory trajectory  # returned interpolated trajectory